backend/app.py

```
@@ -14,7 +14,18 @@
 from databases.qdrant import init_qdrant, store_item_vector, search_similar_items
 from utils import process_item_data, chat_with_database, generate_embedding
 
-app = FastAPI(title="DIY Visual Finder", version="1.0.0")
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.14.1)

Unused function argument: `app`

(ARG001)